examples/illustrations/existence_intervals/existence_intervals.json

```diff
@@ -108,6 +108,42 @@
                 "@value": "2020-01-02T03:04:05.6789Z"
             }
         },
+        {
+            "@id": "kb:Object-bee97e78-8e5e-4ffe-9c2b-28e960f84bbc",
+            "gufo:hasBeginPoint": {
```

Comment:
Here, too, you might want to include the `@type: "gufo:Object"`

Reply:
This will be addressed automatically when `cc10413` gets run through `make`.